test/unit/constants/payment-type.test.js

```diff
@@ -0,0 +1,13 @@
+const { IMMEDIATE, QUARTERLY } = require('../../../app/constants/payment-type')

```

Comment:
This is interesting.

Do we need to test the constant variables?

Reply:
Yes, we need to. If we do not, how do we confirm content if constant is correct?

Reply:
My question is, how do we know `'Immediate payment'` is correct for e.g.?

By their nature are the constant values not thought of to be correct?

Reply:
The constant in question here is provided as a business requirement (Payment Type header in schedule document), hence we need to ascertain the accuracy of what we are giving out as constant against what has been provided as business requirement.

Reply:
Fair, makes senses